app/helpers/application_helper.rb

```diff
@@ -866,6 +866,11 @@ def object_types_for_flash_message(klass, record_ids)
     end
   end
 
+  # condition is result of any boolean condition
+  def li_link_if(args)
+    li_link_if_condition(args.update(:condition => args[:if]))
```

Comment:
@jzigmund : what's the difference between the original `:condition` and the new `:if` ?